extensions/objects/templates/NumberWithUnitsEditor.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+oppia.directive('numberWithUnitsEditor', [
+  '$compile', 'NumberWithUnitsObjectFactory', 'OBJECT_EDITOR_URL_PREFIX',
+  function($compile, NumberWithUnitsObjectFactory, OBJECT_EDITOR_URL_PREFIX) {
+    return {
+      link: function(scope, element) {
+        scope.getTemplateUrl = function() {
+          return OBJECT_EDITOR_URL_PREFIX + 'NumberWithUnits';
+        };
+        $compile(element.contents())(scope);
+      },
+      restrict: 'E',
+      scope: true,
```

Comment:
Hi @vibhor98, can you please remove `scope: true` from this directive and instead use isolated scope? Thanks!